src/test/java/Homework20.java

```diff
@@ -0,0 +1,63 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest {
+
+    @Test
+    @Parameters({"baseURL"})
+    public void deletePlaylist(@Optional String baseURL) throws InterruptedException {
+        // Create a test case deletePlaylist() using @Test annotations
+        // Use the reusable methods
+        // Use @Parameters for passing baseUrl
+        // Given: Navigate to "https://bbb.testpro.io/"
+        // Step1: Log in with your credentials
+        if(baseURL != null) {
+            this.url = baseURL;
+        }
+        login();
+        // Step2: Choose a playlist
+        selectPlaylist();
+        // Step3: Delete the playlist
+        pressDeletePlaylist();
+        // Step4: Validate that the playlist is deleted "Deleted playlist"
+        Assert.assertTrue(getConfirmationPopupText().contains("Deleted playlist"));
+    }
+
+    private void selectPlaylist() throws InterruptedException {
+        WebElement myPlaylist = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        myPlaylist.click();
+        // Thread.sleep(1000);
+    }
+
+    private void pressDeletePlaylist() throws InterruptedException {
+
+        WebElement delPlaylist = driver.findElement(By.cssSelector(".btn-delete-playlist"));
+        delPlaylist.click();
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/200482967-47d312b5-dc6f-4722-ae62-af4b844f9c61.png)

I added `driver.manage().window().maximize();` in the **launchBrowser (BaseTest)**
![image](https://user-images.githubusercontent.com/104761482/200483321-b93343bb-9b33-43e3-b163-4baa4388be0b.png)

Test is running

![image](https://user-images.githubusercontent.com/104761482/200483138-7582aa51-7691-4215-a25e-18f04c39b345.png)